### ISSUE TITLE
repeated lines added to bin/activate

### ISSUE THREAD

**Issue by maniqui** [ekalinin/nodeenv#72]
I've mentioned this as a footnote on issue #43.
If for some reason, you run `nodeenv` more than once, then you end up with duplicated (or triplicated, or...) code on `bin/activate`, once for each time you ran the command.
At least, that's what happens when you create the nodeenv inside `virtualenv` (which, as far as I understand, in that case, both virtualenv and nodeenv share the same file for `activate` command). Not sure if this issue happens when creating a nodeenv outside a virtualenv.

Could it be a solution to keep the nodeenv code in a `activate_nodeenv` file and then `source` it inside virtualenv's `activate`?

**Comment by dnozay** [comment]
I have the same problem in our jenkins job.

```
#!/bin/bash
# This script is used in our continuous integration (jenkins)

set -o errexit
set -o xtrace

# make sure virtualenv is up-to-date
test -d tools || python2.7 -m virtualenv tools
source tools/bin/activate
pip install -r requirements.txt
pip install nodeenv
nodeenv --python-virtualenv

# more stuff ...
```

I've looked at the `activate` script and it kept appending more and more.
https://github.com/ekalinin/nodeenv/blob/master/nodeenv.py#L678-L679

I will fix my script to not call nodeenv if it detects it to have been called previously but if nodeenv was smarter about that, it would be great.

**Comment by ekalinin** [comment]
Should be fixed after #165

**Comment by dnozay** [comment]
👍
